pages/api/internal/assets/[anchor]/[assetType]/index.js

```diff
@@ -114,22 +115,27 @@ export default async function handle(req, res) {
 		}
 	})
 
-	const contract = nft.contract
 
 	// TODO: What to do with assets that aren't attached to any other NFT?
 	// Should this clean up run on an async job?
 	await dropAssetsFromNFT(nft)
 
 	const form = new formidable.IncomingForm()
+	// This is a temporary directory for uploads
+	// It allows us to cache files, but at the same time is isolated from the system's /tmp
+	// this should prevent Arbitrary Code Execution attacks via Image-Uploads.
```

Comment:
I'd drop this comment, it's quite strange to read. Freezing /tmp or wherever the JIT cache is written to during build, so that it cannot be overwritten, prevents arbitrary code execution attacks. Changing the upload directory does not. It's just a necessary consequence of other locations being read-only (and also speeds up uploads a lot by allowing a rename op for completing an upload).

Reply:
Dropped in the course of https://github.com/authenticvision/digitalsoul/pull/41/commits/8807190ea224f95c7a949f5f541f4b70aea8b306